src/pages/Main/Main.js

```diff
@@ -20,7 +20,7 @@ function Main() {
           </div>
           <div className="boxButton">
             <Button
-              action={navigate("/Artist")}
+              action={() => navigate("/Artist")}
```

Comment:
```suggestion
              action={() => navigate("/artist")}
```
처럼 바뀌어야 될 것 같습니다 수정님 pr에 바꿔달라고 요청드렸으니 수정해서 올려주시면 되겠습니다